rails/locale/fr.yml

```diff
@@ -76,11 +76,9 @@ fr:
         other: presque %{count} ans
       half_a_minute: une demi-minute
       less_than_x_minutes:
-        zero: moins d'une minute
```

Comment:
I didn't think about this before but some people might be using these `zero` translations. I need to revert this change and add the equivalent `zero` for the other locales.